app/src/main/java/org/oppia/app/option/AppLanguageActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.app.option
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+private const val PREFERENCE_KEY ="PREFERENCE_KEY"
+
+/** The activity to change the language of the app. */
+class AppLanguageActivity : InjectableAppCompatActivity() {
+  var pref_summary_value: String = ""
+  @Inject
+  lateinit var appLanguageActivityPresenter: AppLanguageActivityPresenter
```

Comment:
This should be directly like this:'
`@Inject lateinit var appLanguageActivityPresenter: AppLanguageActivityPresenter`

Ditto elsewhere.

Reply:
Done.